<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryDialog</class>
 <widget class="QDialog" name="HistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vaccine History Log</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="listView">
     <property name="styleSheet">
      <string notr="true">QListView {
    font-family: Consolas, monospace;
    font-size: 11pt;
    background-color: #f8f9fa;
    border: 1px solid #dee2e6;
    border-radius: 4px;
    padding: 5px;
}
QListView::item {
    padding: 5px;
    border-bottom: 1px solid #dee2e6;
}
QListView::item:hover {
    background-color: #e9ecef;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear History</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #dc3545;
    color: white;
    padding: 8px 16px;
    border-radius: 4px;
    min-width: 100px;
}
QPushButton:hover {
    background-color: #c82333;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #6c757d;
    color: white;
    padding: 8px 16px;
    border-radius: 4px;
    min-width: 100px;
}
QPushButton:hover {
    background-color: #5a6268;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>HistoryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
